xml_inscriptions/transcribed/MA.Glouc.HCM.L.Tmp97.6.16.xml

```diff
@@ -123,7 +123,7 @@
         <div type="edition">
             <ab>
             <lb n="1"/><supplied reason="lost"><expan><abbr>D</abbr><ex>is</ex></expan></supplied> <expan><abbr>M</abbr><ex>anibus</ex></expan>
-             <lb n="2"/><persName><name type="gentilicium" key="Claudia"><supplied reason="lost">Cla</supplied>udiae</name></persName>
+             <lb n="2"/><persName><name type="gentilicium" key="Claudia"><supplied reason="lost">Cla</supplied>udiae</name></persName> con
```

Comment:
should be <expan><abbr>con</abbr><ex>uigi</ex></expan> at the end but I wasn't sure at the time

Reply:
I've gone ahead and incorporated this change.